src/yarn/yarn.yml

```diff
@@ -60,7 +59,6 @@ commands:
   pre-release:
     steps:
       - setup
-      - run: git pull
```

Comment:
Curious, why this line is no longer needed?

Reply:
Well, it actively breaks canary builds because of how circle checkouts work. On a feature branch it seems like the upstream isn’t set appropriately. 

Regardless, it wasn’t really needed to begin with. I added it as a defensive measure because auto makes commits on master. If upstream has been modified it could fail to push due to conflicts. Granted, it will likely fail due to other reasons and will gracefully recover on the next master merge so it seems okay to me.

Reply:
As long as it isn't necessary 👍 from me